### PR TITLE
Update getting-started.md

### PR DESCRIPTION
**Pull request by Bikeman868**
Added an instruction to ignore the paket-files folder. Failing to do this can cause a lot of confusion for first time users of Paket.
I also added a section at the bottom about how to include Paket in the development process when working with other people.

### CODE REVIEW

**Comment by forki** [comment]
Cc @agross

Please also take a look into the FAQ and add questions/answers over there. And: thanks a lot. This is soooo important

File: docs/content/getting-started.md [hunk 1]
```diff
@@ -154,3 +156,22 @@ This command will analyze your
 
 If you are already using NuGet and want to learn how to use the automatic NuGet
 conversion, then read the next [tutorial](convert-from-nuget-tutorial.html).
+
+### Working with Paket
+
+The most frequently used Paket commands are:
+
+* `paket install` which you should run after modifying the `paket.dependencies` file.
```
[the remark this answers is not in the document]
Comment (agross) [reply docs/content/getting-started.md]: And also the files, please link including the "file" part. [Example](https://fsprojects.github.io/Paket/getting-started.html#Specifying-dependencies)

File: docs/content/getting-started.md [hunk 2]
```diff
@@ -154,3 +156,22 @@ This command will analyze your
...
+  This command will update the `paket.lock` file, then update all of your Visual Studio
+  projects to the new versions.
+
+* `paket restore` which you should run after cloning the repository or switching branches.
+  This command will take the current `paket.lock` file and update your Visual Studio projects
+  so that they are referencing the correct versions of NuGet packages. If you are using a
+  continuous integration system like TeamCity or Jenkins then you should also run this
+  command after getting the source code and before compiling it.
+  
+* `paket update` which you can run to update your project to the latest versions of all
```
Comment (agross) [inline docs/content/getting-started.md]: project*s*

**Comment by agross** [comment]
Sorry, I have been on vacation. Looks good, but please amend as per my comments. Many thanks!

**Comment by Bikeman868** [comment]
I updated my fork as per your comments.
Do I need to create a new pull request, or is my new check in visible here for you?

**Comment by Bikeman868** [comment]
I did look at the FAQ and there are a number of things that I could add that might help others, unfortunately I do not know enough about what Paket does to be able to write them, and nowhere in the documentation does it describe what Paket does either.
By this I mean that there is no documentation on what files are modified by Paket and in what ways they are changed by the various commands. I can not figure this out by observing its behaviour and it is not documented anywhere, so for now I rolled our solution back to using the NuGet Package Manager.
I like the design principles employed by Paket and will give it another try when I have more time.

**Comment by agross** [comment]
> or is my new check in visible here for you?

This PR was updated automatically after you pushed to the same branch that you created the PR from. All good.

> By this I mean that there is no documentation on what files are modified

Does the list below "DOCUMENTATION" on the right side of [this page](https://fsprojects.github.io/Paket/index.html) help to answer your questions?

> I rolled our solution back to using the NuGet Package Manager

It's unfortunate that you saw the need to roll back. Does this help to resolve your questions? [1](https://cockneycoder.wordpress.com/2017/08/07/getting-started-with-paket-part-1/) [2](https://cockneycoder.wordpress.com/2017/08/14/migration-basics-from-nuget-to-paket/) [3](https://cockneycoder.wordpress.com/2017/08/21/migrating-complex-nuget-solutions-to-paket/) [4](https://cockneycoder.wordpress.com/2017/09/04/value-added-features-with-paket/)

**Comment by Bikeman868** [comment]
I already read all of the documentation including the page you referenced before making my assertion that nowhere in the documentation does it describe what Paket actually does. This is a common problem, where the developers that built the solution are so intimately familiar that they can not see where they are making assumptions that are not at all obvious to someone coming to it cold.

I had to roll back because our continuous integration build had been broken for the entire day and I couldn't justify continuing with it in a broken state for any longer.

It is my intention to have another go at integrating Paket in the future and next time I will create a new sandbox environment in the continuous integration system so that others are not affected.

**Comment by Bikeman868** [comment]
This is an example of what I think is missing.

This documents (to the best of my knowledge) what happens when the continuous integration system builds a solution that uses Paket.

Documenting these scenarios is way more useful for newcomers than documenting each command but giving no indication of how the commands should be used together.

continuous integration build steps

**Pull request merged**